driver/src/solution_submission/mod.rs

```diff
@@ -1,16 +1,13 @@
 #![allow(clippy::ptr_arg)] // required for automock
 
 use crate::contracts::stablex_contract::StableXContract;
-use crate::error::DriverError;
 use crate::models::{Order, Solution};
 
+use anyhow::Result;
+use ethcontract::U256;
 #[cfg(test)]
 use mockall::automock;
 
-use ethcontract::U256;
-
-type Result<T> = std::result::Result<T, DriverError>;
```

Comment:
Does this have something to do with the awkward `Result<SometimesErrorSometimesNot>`?